src/main/java/org/ua/javaPro/berezhnoy/bank/currencyConverterAPI/CurrencyConfig.java

```diff
@@ -0,0 +1,21 @@
+package org.ua.javaPro.berezhnoy.bank.currencyConverterAPI;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CurrencyConfig {
+    @Bean
+    @ConditionalOnProperty(name = "currency.converter.provider", havingValue = "dummy", matchIfMissing = true)
```

Comment:
```suggestion
    @ConditionalOnProperty(name = "currency.converter.provider", havingValue = "dummy")
```
не логічно dummy використовувати за замовчуванням.